SNIClient.py

```diff
@@ -1131,6 +1129,126 @@ async def main():
     if input_task:
         input_task.cancel()
 
+def get_alttp_settings(romfile: str):
+    lastSettings = Utils.get_adjuster_settings(GAME_ALTTP)
+    adjusted = False
+    adjustedromfile = ''
+    if lastSettings:
+        choice = 'no'
+        if not hasattr(lastSettings, 'auto_apply') or 'ask' in lastSettings.auto_apply:
+
+            whitelist = {"music", "menuspeed", "heartbeep", "heartcolor", "ow_palettes", "quickswap",
+                        "uw_palettes", "sprite", "sword_palettes", "shield_palettes", "hud_palettes",
+                        "reduceflashing", "deathlink"}
+            printed_options = {name: value for name, value in vars(lastSettings).items() if name in whitelist}
+            if hasattr(lastSettings, "sprite_pool"):
+                sprite_pool = {}
+                for sprite in getattr(lastSettings, "sprite_pool"):
+                    if sprite in sprite_pool:
```

Comment:
I think you might want collection.Counter(last_settings.sprite_pool), which is a subclass of a dict

Reply:
Same as previous, not my code, not sure what the original intention of that code is.